int16 node_id
std_msgs/Int32MultiArray indices
---
std_msgs/Int32MultiArray indices
std_msgs/Int32MultiArray data
bool success